fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+
+  desc "Run test using fast lane"
+  lane :tests do 
+    run_tests(scheme: "kwarchart")
+  end
+  
+  desc "Fastlane status"
+  lane :fast do 
+    ensure_git_status_clean
+    update_fastlane
```

Comment:
We don't need to update fastlane, that's the reason why it's updating the gemfile lock